plugin/src/main/kotlin/com/louiscad/splitties/PluginConfig.kt

```diff
@@ -0,0 +1,8 @@
+package com.louiscad.splitties
+
+
+object PluginConfig {
+    const val GROUP = "com.louiscad.splitties"
+    const val GRADLE_PROPERTY = "version.$GROUP"
+    const val SPLITTIES_VERSION = "2.1.1"
```

Comment:
Shouldn't this point to the latest alpha release?

Reply:
done